static/app/components/projects/bookmarkStar.tsx

```diff
@@ -66,10 +66,7 @@ const Star = styled(IconStar, {shouldForwardProp: p => p !== 'isBookmarked'})<{
   isBookmarked: boolean;
 }>`
   color: ${p => (p.isBookmarked ? p.theme.yellow300 : p.theme.gray200)};
-
-  &:hover {
-    color: ${p => (p.isBookmarked ? p.theme.yellow200 : p.theme.gray300)};
-  }
```

Comment:
Does the hover look weird now?

Reply:
I'll probably add more of a universal animation for the hover effect on favorites